tests/linearizability/traffic.go

```diff
@@ -33,7 +33,7 @@ type Traffic interface {
 type putGetTraffic struct{}
 
 func (t putGetTraffic) Run(ctx context.Context, c *recordingClient, limiter *rate.Limiter) {
-	maxOperationsPerClient := 1000000
+	maxOperationsPerClient := 10
```

Comment:
Any reason to set this? This number is used to ensure each client write has unique value. This is done by shifting starting id value for each client and limiting number of writes. For example first client writes values `0...1000000`, second client writes `1000000...2000000` and so forth. 

Lowering this number just results in clients making 10 requests and finishing.